README.md

```diff
@@ -16,26 +16,40 @@ Sandbox's client-side APIs.
 ## Bidding and Auction services
 
 * [Bidding and Auction services](bidding_auction_services_api.md)
+* [Bidding and Auction services: Protected App Signals Ad Selection](bidding_auction_services_protected_app_signals.md)
 * [Bidding and Auction services onboarding and self-serve guide](bidding_auction_services_onboarding_self_serve_guide.md)
 * [Bidding and Auction services payload optimization](https://github.com/privacysandbox/fledge-docs/blob/main/bidding-auction-services-payload-optimization.md)
 * [Bidding and Auction services system design](bidding_auction_services_system_design.md)
 * [Bidding and Auction services multi seller auctions](https://github.com/privacysandbox/fledge-docs/blob/main/bidding_auction_services_multi_seller_auctions.md)
-* [Bidding and Auction services AWS cloud support and deployment guide](bidding_auction_services_aws_guide.md)
-* [Bidding and Auction services GCP cloud support and deployment guide](bidding_auction_services_gcp_guide.md)
 * [Protected Audience event level reporting with Bidding and Auction services](bidding_auction_event_level_reporting.md)
+* Cloud support: [Bidding and Auction services AWS cloud support and deployment guide](bidding_auction_services_aws_guide.md)
+* Cloud support: [Bidding and Auction services GCP cloud support and deployment guide](bidding_auction_services_gcp_guide.md)
 
 ## Key/Value service
 
 * [Key/Value service trust model](key_value_service_trust_model.md)
-* [Key/Value service user-defined functions (UDFs)](key_value_user_defined_functions.md)
+* [Key/Value service use cases](key_value_service_use_cases.md)
+* [Key/Value service user-defined functions (UDFs)](key_value_service_user_defined_functions.md)
 * [Key/Value sharding](key_value_sharding.md)
 
+## Ad Retrieval service
+
+* [Protected App Siganls: Ad Retrieval service](ad_retrieval_overview.md)
+
+## Inference
+
+* Bidding and Auction services: Inference Overview](inference_overview.md)
```

Comment:
missing [  (so the link does not render as a link / work)

Reply:
Fixed.